app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -363,7 +363,7 @@ def handle_successful_status_update(enrollment, response)
       reason: 'Successful status update',
       job_name: self.class.name,
     )
-    enrollment.profile.activate_after_passing_in_person
+    enrollment.profile&.activate_after_passing_in_person
     enrollment.update(
```

Comment:
I'm probably over thinking this, but do we want to update the enrollment if the profile is nil? Like you mention, it's kinda nonsense, but I guess it makes sense to have some sort of record?

Reply:
Good question!

In LG-12050, I had edited the user's enrollment to "cancelled" status to stop it from failing every hour. It made sense to me because the operation was overall a failure, and I was sort of administratively "cancelling" the operation.

But, it felt more accurate to let it update it to "passed" here since that _is_ what we get from USPS. I don't feel strongly at all, though.